packages/pharos/src/components/dropdown-menu/pharos-dropdown-menu-item.ts

```diff
@@ -56,6 +57,13 @@ export class PharosDropdownMenuItem extends ScopedRegistryMixin(FocusMixin(Pharo
   @property({ type: String, reflect: true })
   public target: '_blank' | '_parent' | '_self' | '_top' = '_self';
 
+  /**
+   * Indicates the type of link.
```

Comment:
Perhaps more accurately, quoting more or less from [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel):

```suggestion
   * Indicates the relationship of the resource to the current document.
```